win32/Cargo.toml

```diff
@@ -12,6 +12,7 @@ x86 = { workspace = true, optional = true }
 anyhow = "1.0"
 bincode = "1.3.3"
 bitflags = "1.3.2"
+chrono = "0.4.38"
```

Comment:
Do we need this at the win32 layer?  In particular it's additional code on web.

Reply:
Yeah, [SYSTEMTIME](https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-systemtime) means we need _some_ code that understands calendar dates and can convert them. Open to alternate ideas, but I don’t see an easy way to avoid a dep like this.